cfme/cloud/keypairs.py

```diff
@@ -88,8 +82,7 @@ def is_displayed(self):
             self.entities.title.text == 'Key Pairs')
 
     toolbar = View.nested(KeyPairToolbar)
-    entities = View.nested(KeyPairEntities)
-    paginator = PaginationPane()
+    including_entities = View.include(VMEntities, use_parent=True)
```

Comment:
@mshriver, why do you use VMEntities, not BaseEntitiesView here ?

Reply:
Discussed in depth offline, but the core of it was I misunderstood the need to version pick the entity class with the new entities - this is only necessary when implementing data property, which isn't needed here.

Testing with BaseEntitiesView included.